### PR TITLE
Add some random translation tags.

### PR DESCRIPTION
**Pull request by amotin**
I've completed Russian translation as of now. Yay!

So everything that still remains in English in UI probably misses respective tags. I've tried to add some in this PR, but don't know whether it is correct, and it is definitely far from all.  At least there are folder headers in iSCSI are not translated.  And I haven't checked, but I guess that enclosure management may be also.

### CODE REVIEW

File: src/app/pages/system/update/update.component.html [hunk 1]
```diff
@@ -95,33 +95,35 @@
 </mat-card>
 
 <mat-card *ngIf="updates_available && pre_release_train && isfreenas" id="stable-warning-card">
-  <div class="stable-warning"><i class="material-icons" id="update-info">info</i><strong>This is not a production release, and should only be used for testing.</strong></div>
-  <div class="stable-warning" id="stable_warning_line_2">Before updating, please read the
-    <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">release notes</a>.
+  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>
+    <strong>{{"This is not a production release, and should only be used for testing." | translate}}</strong>
+  </div>
+  <div class="stable-warning" id="stable_warning_line_2">{{"Before updating, please read the
```
Comment (aervin) [inline src/app/pages/system/update/update.component.html]: I think you'll need to split this into two translated strings:
```html
<div class="stable-warning" id="stable_warning_line_2">
  {{"Before updating, please read the" | translate }}
  <!-- whitespace character -->
  &nbsp;
  <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">{{ "release notes" | translate }}</a>.
</div>
```

File: src/app/pages/system/update/update.component.html [hunk 2]
```diff
@@ -95,33 +95,35 @@
...
+    <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">release notes</a>." | translate}}
   </div>
 </mat-card>
 
 <mat-card *ngIf="updates_available && release_train && isfreenas" id="stable-warning-card">
-  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>Before updating, please read the
-    <a href='https://www.ixsystems.com/blog/knowledgebase_category/freenas/?latest=true' target="new">release notes</a>.
+  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>{{"Before updating, please read the
```
Comment (aervin) [inline src/app/pages/system/update/update.component.html]: as above

File: src/app/pages/system/update/update.component.html [hunk 3]
```diff
@@ -95,33 +95,35 @@
...
+    <a href='https://www.ixsystems.com/blog/knowledgebase_category/freenas/?latest=true' target="new">release notes</a>." | translate}}
   </div>
 </mat-card>
 
 <mat-card *ngIf="updates_available && pre_release_train && !isfreenas" id="stable-warning-card">
-  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>Before updating, please read the
-    <a href='https://www.ixsystems.com/blog/knowledgebase_category/truenas/?latest' target="new">release notes</a>.
+  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>{{"Before updating, please read the
```
Comment (aervin) [inline src/app/pages/system/update/update.component.html]: as above

File: src/app/pages/system/update/update.component.html [hunk 4]
```diff
@@ -95,33 +95,35 @@
...
+    <a href='https://www.ixsystems.com/blog/knowledgebase_category/truenas/?latest' target="new">release notes</a>." | translate}}
   </div>
 </mat-card>
 
 <mat-card *ngIf="updates_available && release_train && !isfreenas" id="stable-warning-card">
-  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>Before updating, please read the
-    <a href='https://www.ixsystems.com/blog/knowledgebase_category/truenas/?latest' target="new">release notes</a>.
+  <div class="stable-warning"><i class="material-icons" id="update-info">info</i>{{"Before updating, please read the
```
Comment (aervin) [inline src/app/pages/system/update/update.component.html]: as above

File: src/app/pages/system/update/update.component.html (hunk 1 above)
Comment (amotin) [reply src/app/pages/system/update/update.component.html]: Splitting single sentence into two parts makes correct translation difficult.  I suppose the problem is in nested {{ }}?  Is there any syntax to express it without splitting?

File: src/app/pages/system/update/update.component.html (hunk 2 above)
Comment (amotin) [reply src/app/pages/system/update/update.component.html]: Here I don't see a technical reason other then URL appearing in translated string. But I saw that practice widely in the translation already.

File: src/app/pages/system/update/update.component.html (hunk 1 above)
Comment (aervin) [reply src/app/pages/system/update/update.component.html]: Perhaps the simplest solution would be to make the full sentence an anchor link?
```html
<div class="stable-warning" id="stable_warning_line_2">
  <a href='https://www.ixsystems.com/blog/library/{{train_version}}' target="new">
    {{ "Before updating, please read the release notes." | translate }}
  </a>
</div>
```

File: src/app/pages/system/update/update.component.html (hunk 2 above)
Comment (aervin) [reply src/app/pages/system/update/update.component.html]: this was an error, sorry

**Pull request merged**